### ISSUE TITLE
abandon support for WPF, Cocoa

### ISSUE THREAD

**Issue by lytico** [dotdevelop/community#6]
it's hard and seems to be beyond the scope of this project to have 3 GUI backends.

As Gkt is already supported on Linux, Windows and MacOs, it's a luxery to provide 2 more backends.

To my opinion, this was the reason that MS abandoned linux and windows support, as it got too complicated.

**Comment by lextm** [comment]
The reasons for Microsoft/Xamarin to abandon Windows and Linux port of MonoDevelop can be rather complicated and I don't think on technical side.

**Comment by trampster** [comment]
Given our resources are rather limited, I think we should focus on GTK.

**Comment by lextm** [comment]
@trampster It is reasonable for the community to focus on GTK side, but I don't think now isn't the time to "abandon" WPF/Cocoa because Microsoft/Xamarin might continue their work on that side, as long as they are willing to make the source code available for integration. At least the Cocoa port is highly active as part of VS for Mac.

**Comment by lytico** [comment]
ms abandoned open source monodevelop
there will not be any source code available for integration

**Comment by lytico** [comment]
VS for Mac is closed source now

**Comment by lextm** [comment]
@lytico I will try to ping Miguel via personal channel and see if more information can be collected than "it won't".

If Microsoft does have a plan to turn away from an open source MonoDevelop, then beyond a community fork, the community should also try to take over other assets (the brand, the homepage and so on). There should be a possibility to prepare a handover, than asking the community to pick up another name/icon.

**Comment by lytico** [comment]
mono will be abandoned. it's merged into .net 5. so maybe it's time for new brand, like the rebranding from SharpDevelop to MonoDevelop. 
Another brand could be back to SharpDevelop. It's out of MS and people behind maybe support us. Maybe contact them? I don't know.

**Comment by lytico** [comment]
btw, contacting miguel is always a good idea.

**Comment by trampster** [comment]
I'm doubtful that the MonoDevelop project will be processing any new merge requests, we could approach Microsoft and ask them if they would gift us the repo and branding. However I also see this as unlikely. 

The only reason we know the project is abandoned at all is because we pressured Miguel. And Miguel told us he is no longer in charge of the project. 

It is likely he doesn't have the authority to give us permission to use the repo or branding.

And the people who are in charge chose to first break the repo then abandon it without any announcement. I don't think we can rely on Microsoft for anything.

**Comment by lytico** [comment]
no, i would never pressure miguel. it was a kind request, and we got kind answers.

**Comment by lextm** [comment]
@trampster Microsoft did free several open source projects (even Windows Live Writer), so I won't consider freeing MonoDevelop a mission impossible.

Several assets (monodevelop.com, MonoDevelop brand and so on) are critical for the community to continue this open source project. Before Microsoft gives a clear no, we should try our best.

**Comment by trampster** [comment]
To be honest when Miguel said what he did, and then said he was no longer in charge of the project, he was probably putting his neck out, In a large company like Microsoft you don't make public announcements like that when you are not in change. Doing so could have harmed his career.

**Comment by trampster** [comment]
@textm, Yes I agree, we should at least ask.

**Comment by lextm** [comment]
@trampster Since it is a big company, it is just challenging to find the correct contact, but I am already getting started on the hunt.

**Comment by trampster** [comment]
However if the answer is silence rather then no, then we should be prepared to do our own thing.

**Comment by lextm** [comment]
@trampster communication with Microsoft can come in parallel. We only need an answer before finally making an "official" community based release.
